frontend/src/Page/index.js

```diff
@@ -133,6 +133,21 @@ function Page(props) {
     authors = <Meta title={title}>{authorLinks}</Meta>;
   }
 
+  let custom = [];
+
+  for (let title in props.custom) {
```

Comment:
The code block basically transforms each item in `props.custom` in a corresponding new item. Using `Array.map` will better express this procedure.

Reply:
`props.custom` is an object, so `Array.map` is not an option, if I’m not mistaken …

Reply:
`Object.entries()` is your friend https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries